src/crypto/tls/delegated_credentials_test.go

```diff
@@ -446,7 +462,9 @@ func testServerGetCertificate(ch *ClientHelloInfo) (*Certificate, error) {
 	}
 
 	if versOk && ch.SupportsDelegatedCredential {
-		return dcTestCerts["dcP256"], nil
+		serverCert := dcTestCerts["dcP256"]
+		serverCert.DelegatedCredentials = serverDC[dcCount:]
```

Comment:
(Note for my understanding: `initDCTest` resets `dcTestCerts` and `dcTestCerts["dcCertP256"]` which is why it is okay for `testServerGetCertificate` to have side-effects such as overwriting `dcTestCerts["dcP256"].DelegatedCredentials`.)

I was wondering why `serverDC[dcCount:]` was used. Would `serverDC[dcCount:dcCount+1]` be sufficient to limit the test to a specific DC? Or is there a reason to include more than one?

Reply:
It is more for testing with each specific DC and algorithm (a DC with ed25519, a Dc with eddsa256, etc). I'll prefer if the tests were only using one DC (it simplifies all) but is was requested by reviewers to test with them all.

Reply:
So based on a private chat with Sofía, it appears that it deliberately includes more than one to test this code path:
https://github.com/cloudflare/go/blob/1777b4c8ee76d5bc3e230d21fe3539d9cb565e33/src/crypto/tls/handshake_server_tls13.go#L464-L472